### ISSUE TITLE
[QUESTION ]Spyware Filter and/or EasyPrivacy

### ISSUE THREAD

**Issue by BPower0036** [AdguardTeam/AdguardFilters#77015]
I use both Spyware Filter and the EasyPrivacy filter.
However if EasyPrivacy uses all its rules the AdGuard Spyware Filter uses only the half of its rules.
So there are a lot of double filter rules.

My question is: which one should I use?
Only EasyPrivacy or only Spyware filter.

P.S. I use also the AdGuard DNS Filter and Peter Lowe’s Ad and tracking server list

**Comment by Yuki2718** [comment]
@BPower0036 Yes, lots of duplicates and redundant filters if you use both, the half is quite underestimate and not all redundant rules can easily be figured out due to different filter styles. Generally, EP blocks bit more *on average* thx to its aggressive use of generic rules, but this is a double-edged sword which has also been causing more troubles aka false positive. I emphasize "on average", it's finally depends on what sites you visit and there are some regional biases too, EP is relatively strong on Western Europe while AGTPF on Eastern Europe and Eastern Asia. I generally don't recommend to use both - few benefit for higher chance of trouble.
